automation/vars/main.yml

```diff
@@ -553,6 +553,8 @@ wal_g_patroni_cluster_bootstrap_recovery_conf:
   - recovery_target_action: "promote"
   - recovery_target_timeline: "latest"
 #  - recovery_target_time: "2020-06-01 11:00:00+03"  # Point-in-Time Recovery (example)
+wal_g_prefetch_dir_create: false # or 'true'
+wal_g_prefetch_dir_path: "{{ postgresql_home_dir }}/wal-g-prefetch"
```

Comment:
Maybe it makes sense to enable this by default?

And also add `WALG_PREFETCH_DIR` to the configuration example and to [auto_conf](https://github.com/vitabaks/autobase/blob/2.1.0/automation/roles/wal-g/tasks/auto_conf.yml) tasks.